### PR TITLE
Fix: Bundlesize CI check

### PR DESCRIPTION
**Pull request by 3nvi**
## Background

Our CI checks are failing, cause one of our bundles are a bit too big. This PR makes sure to split some assets into separate chunks in order to have a faster loading

## Changes

- Split `mixpanel` into its own chunk
- Merge telemetry `.env` files
- rename `loggers.ts` to `errors.ts`
- fix `lodash-es` for `lodash`

## Testing

- Manually verified 
- `npm run test`

### CODE REVIEW

File: web/.env.telemetry [hunk 1]
```diff
@@ -1,3 +1,4 @@
+MIXPANEL_PUBLIC_TOKEN=1cd3d6e5e0b3f8085e437fb14a26695f
```
Comment (3nvi) [inline web/.env.telemetry:1]: merged  tellemetry into a single `.env` for better management

File: web/src/helpers/analytics.ts [hunk 2]
```diff
@@ -17,25 +17,30 @@
...
-  window.requestIdleCallback(() => {
-    try {
-      mx.init(mixpanelPublicToken);
-      mx.track(...args);
-    } catch (e) {
-      // Reporting to sentry
-      logError(e);
-    }
+
+  import(/* webpackChunkName: "mixpanel" */ 'mixpanel-browser').then(mx => {
```
Comment (alexmylonas) [inline web/src/helpers/analytics.ts:31]: Tested seems ok!

**Review by alexmylonas — APPROVED** [review]
LGTM!

**Review by vorillaz — APPROVED** [review]
📈

**Pull request merged**